xml/admin_administration.xml

```diff
@@ -137,6 +137,52 @@ cmd.run "<replaceable>df -h</replaceable>"</command>
    Replace the example <command>df -h</command> with a command of your choice.
    The output will be produced in your current terminal session.
   </para>
+
+  <sect2 xml:id="sec.admin.salt.workers">
+   <title>Adjusting The Number Of Salt Workers</title>
+   <para>
+    It will sometimes be necessary to resize the cluster to adjust for workloads
```

Comment:
`resize the cluster` might indicate changing the number of masters/workers on the cluster. Maybe a rewording here would be better: `It will sometimes be necessary to increase the number of worker threads of the salt master`.

Reply:
By "resizing the cluster" i mean adding nodes to the cluster that then make adjusting the worker threads necessary.

Reply:
I see, that makes sense. Maybe a `Kubernetes cluster` in this case would be nice.

Reply:
added